lib/grainstore/index.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
         return version;
     },
     MMLStore: require('./mml_store'),
-    MMLBuilder: require('./mml_builder')
+    MMLBuilder: require('./mml-builder/mml-builder-worker')
```

Comment:
`mml_builder-worker` extends from `mml_builder` so the current interface is respected.